fineract-provider/src/main/resources/META-INF/persistence.xml

```diff
@@ -28,6 +28,8 @@
 	<persistence-unit name="jpa-pu" transaction-type="RESOURCE_LOCAL">
 		<provider>org.apache.openjpa.persistence.PersistenceProviderImpl</provider>
 	<properties>
+		<property name="openjpa.jdbc.EagerFetchMode" value="none"/>
+		<property name="openjpa.jdbc.SubclassFetchMode" value="none"/>
```

Comment:
These two OpenJPA configuration changes here (`EagerFetchMode` & `SubclassFetchMode`) strike me as major data access changes that perhaps should not be "hidden" within this PR.. would you be willing to raise a new JIRA and a separate PR and remove this from here so that it can be reviewed and discussed in its own right?

Reply:
i agree with @vorburger

Reply:
Comment has been addressed and new PR has been created: https://github.com/apache/fineract/pull/777